### PR TITLE
added tags parameter in azurev3

### PR DESCRIPTION
**Pull request by pripatra**
_This is the default pull request template. You can customize it by adding a `pull_request_template.md` at the root of your repo or inside the `.github` folder._

# Jira Ticket

_Include a link to your Jira Ticket_
_Example: [JIRAISS-1234](https://spotinst.atlassian.net/browse/JIRAISS-1234)_

# Demo

_Please add a recording of the feature/bug fix in work. if you added new routes, the recording should show the request and response for each new/changed route_

# Checklist:
- [ ] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [ ] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have validated all the requirements in the Jira task were answered
- [ ] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**)

### CODE REVIEW

File: service/elastigroup/providers/azure/v3/azure.go [hunk 1]
```diff
@@ -189,6 +190,14 @@ type ManagedServiceIdentity struct {
 	nullFields      []string
 }
 
+type Tag struct {
```
Comment (chandra1-n) [inline service/elastigroup/providers/azure/v3/azure.go:193]: use Tags

File: service/elastigroup/providers/azure/v3/azure.go [hunk 2]
```diff
@@ -627,6 +636,13 @@ func (o *LaunchSpecification) SetShutdownScript(v *string) *LaunchSpecification
...
 }
 
+func (o *LaunchSpecification) SetTags(v []*Tag) *LaunchSpecification {
+	if o.Tags = v; o.Tags == nil {
+		o.nullFields = append(o.nullFields, "Tags")
+	}
+	return o
+}
+
 // endregion
```
Comment (chandra1-n) [inline service/elastigroup/providers/azure/v3/azure.go:646]: setKey and setValue functions are missing

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the review comments

File: service/elastigroup/providers/azure/v3/azure.go [hunk 3]
```diff
@@ -85,6 +85,7 @@ type LaunchSpecification struct {
 	Login                    *Login                    `json:"login,omitempty"`
 	CustomData               *string                   `json:"customData,omitempty"`
 	ManagedServiceIdentities []*ManagedServiceIdentity `json:"managedServiceIdentities,omitempty"`
+	Tags                     []*Tag                    `json:"tags,omitempty"`
```
[the remark this answers is not in the document]
Comment (pripatra) [reply service/elastigroup/providers/azure/v3/azure.go]: Done

File: service/elastigroup/providers/azure/v3/azure.go (hunk 1 above)
Comment (pripatra) [reply service/elastigroup/providers/azure/v3/azure.go]: Done

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**